### DISCUSSION TITLE
Is there a way to direct connect?

### DISCUSSION THREAD

**Discussion by panpanpanpaan** [andyholmes/valent#582] [Help]
Hi!, im trying to connect my droidian phone with my pc, and nether the pc or the phone detect each other, this is a common issue in my house cause i do have 3 routers, and the pc is connected to the main one, i can't really choose either connect to any of the routers in my phone, so it would be useful to be able to connect with an ip address or something like that

**Comment by andyholmes** [comment]
At the moment there's no way to connect to a device by IP. I'm surprised that the newer mDNS support doesn't solve these odd networking problems, though :thinking: 

Is it possible that mDNS/DNS-SD traffic is blocked on your local network?

**Reply by panpanpanpaan** [reply]
how can i check if its blocked?

**Reply by andyholmes** [reply]
I guess you should check your routers and firewalls for mDNS/DNS-SD (usually port 5353).
